### ISSUE TITLE
Pair-programming on a project

### ISSUE THREAD

**Issue by alepez** [alepez/devand#58]
If I want to pair-program on a particular opensource project, I would like to propose/select the project (it can be a github handle or a full url).

**Comment by VincentFoulon80** [comment]
Maybe add the possibility to attach projects on the user profile or even "project category" like games, system, websites...  So that we could also match the same interests ?

**Comment by alepez** [comment]
Good idea! This is related to #41 (but instead of framework we can just say "category" or "tag"). Tags may be similar to [StackOverflow tags](https://stackoverflow.com/tags)
